change/@fluentui-react-toast-560606fc-9d0b-489e-9ffe-4f82be76a305.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Feat: Updated Toast to use semantic tokens",
```

Comment:
Hmmm can you be a bit more specific here? I think actually this should read "fix: Exporting ToastContainer and related functions"